torch/nn/modules/conv.py

```diff
@@ -1203,6 +1209,12 @@ def __init__(
         if bias:
             self.bias = UninitializedParameter(**factory_kwargs)
 
+    def _get_in_channels(self, input: Tensor) -> int:
```

Comment:
nit: all of these are the same except the number of expected dimensions and function name. Should this be in a helper function like the c++ case?